src/ui/util/s3.test.ts

```
@@ -0,0 +1,212 @@
+import { describe, expect, test, vi, beforeEach, afterEach } from "vitest";
+import { uploadToS3PresignedUrl, downloadFromS3PresignedUrl } from "./s3";
```

ESLint

[error] Missing file extension for "./s3"

(import/extensions)